src/sqlfluff/utils/reflow/reindent.py

```diff
@@ -1309,20 +1353,24 @@ def lint_line_length(
 
             # Lastly deal with the "normal" case.
             else:
-                # As a blunt solution, force indents at the lowest available
-                # option.
-                reflow_logger.debug("    Handling as normal line.")
+                # For now, the algorithm we apply isn't particularly elegant
+                # and just finds the "outermost" opportunity to add additional
+                # line breaks and adds them. It doesn't add more them at more
+                # levels or selectively to add indents preferentially to an
```

Comment:
Can you review these two lines? This part reads strangely:
>add more them at more
>levels or selectively to add indents

Reply:
Yeah I'll rethink. It's correct, but unclear.